plugins/jobtype/test/azkaban/jobtype/TestHadoopTuningConfigurationInjector.java

```diff
@@ -0,0 +1,67 @@
+package azkaban.jobtype;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.junit.Assert;
+import org.junit.Test;
+
+import azkaban.flow.CommonJobProperties;
+import azkaban.utils.Props;
+
+import com.google.common.io.Files;
+
+
+public class TestHadoopTuningConfigurationInjector {
+
+  @Test
+  public void testTuningErrorHandler() throws IOException {
+
+    File root = Files.createTempDir();
+
+    Props props = new Props();
+    props.put(HadoopConfigurationInjector.INJECT_PREFIX + "mapreduce.task.io.sort.mb", "100");
+    props.put(HadoopConfigurationInjector.INJECT_PREFIX + "mapreduce.task.io.sort.factor", "100");
+    props.put(HadoopConfigurationInjector.INJECT_PREFIX + "mapreduce.reduce.memory.mb", "2048");
+    props.put(HadoopConfigurationInjector.INJECT_PREFIX + "mapreduce.map.memory.mb", "2048");
+
+    props.put(CommonJobProperties.EXEC_ID, CommonJobProperties.EXEC_ID);
```

Comment:
Could use a loop here to eliminate a lot of code

Reply:
Probably create an array of all of these props you want to add, iterate through that array here and put them into `props`, then iterate through it again at the end and verify that it hasn't changed

Reply:
Done.